src/js/views/patients/worklist/action-item.hbs

```diff
@@ -1,7 +1,7 @@
 <td class="table-list__cell table-list__item-check js-no-click"><button class="button--checkbox js-select">{{#if isSelected}}{{fas "check-square"}}{{else}}{{fal "square"}}{{/if}}</button></td>
 <td class="table-list__cell w-15">
-  <span class="worklist-list__patient-sidebar-icon js-patient-sidebar-button">
-    <button class="button-secondary--compact">
+  <span class="worklist-list__patient-sidebar-icon">
+    <button class="button-secondary--compact js-patient-sidebar-button">
```

Comment:
I moved the `.js-patient-sidebar-button` to the button here. Seems to make more sense that the trigger class name is on the button instead of the span.

Did the same in the `flow-item.hbs` file too.